test/heterogeneous_ice_nucleation_tests.jl

```diff
@@ -69,8 +60,24 @@ function test_heterogeneous_ice_nucleation(FT)
                 Si_med,
                 T_warm,
             )
+            TT.@test CMI_het.MohlerDepositionRate(
+                dust,
+                ip.deposition,
+                Si_med,
+                T_cold,
+                dSi_dt,
+                N_aer,
+            ) > CMI_het.MohlerDepositionRate(
+                dust,
+                ip.deposition,
+                Si_med,
+                T_warm,
+                dSi_dt,
+                N_aer,
+            )
```

Comment:
Would it be possible to have a test that compares the rate and AF Mohler parameterizations? Are there any conditions where they should give similar output (for a non-zero Si case)